Add PHPStan checks (level 1)

cedric-anne opened the pull request:
Finalization of #37 based on shared workflow (https://github.com/glpi-project/plugin-ci-workflows/blob/v1/.github/workflows/continuous-integration.yml)

trasher commented on .github/workflows/continuous-integration.yml:
```diff
@@ -22,50 +21,9 @@ jobs:
-          vendor/bin/parallel-lint --colors --checkstyle --exclude ./vendor/ . | vendor/bin/cs2pr
-      - name: "PHP_CodeSniffer"
-        run: |
-          vendor/bin/phpcs -q --report=checkstyle | vendor/bin/cs2pr
```
This one seems not running

cedric-anne replied:
They are, but as output is piped to `cs2pr`, it is actually hard to know it. I will fix this.

trasher reviewed (changes requested):
phpstan is not run

cedric-anne commented:
I removed usage to `cs2pr` for now and added some messages (in yellow in following screenshots).
![image](https://github.com/pluginsGLPI/oauthimap/assets/33253653/d142fc13-e8c2-4452-ada7-c8aae6b49e52)
